views/PlaylistVideoView.js

```diff
@@ -0,0 +1,18 @@
+var PlaylistVideoView = Backbone.View.extend({
+  tagName: 'li',
+
+  className: 'row video',
+
+  template: Handlebars.compile($('#playlist-video-template').html()),
+
+  render: function() {
+    var modelIndex = this.model.collection.indexOf(this.model);
```

Comment:
What is this?  Why aren't you just telling the view what model to render rather than it searching for it.  Your views are WAY too smart.